src/components/UserProfile/ProfileHeader/index.tsx

```diff
@@ -65,11 +65,13 @@ const ProfileHeader: React.FC<ProfileHeaderProps> = ({
                 {user.displayName}
               </a>
             </Link>
-            {user.email && (
-              <span className="text-sm text-gray-400 sm:text-lg sm:ml-2">
-                ({user.email})
-              </span>
-            )}
+            {user.displayName.toLowerCase() !== user.email.toLowerCase() &&
+              (user.id === loggedInUser?.id ||
+                hasPermission(Permission.MANAGE_USERS)) && (
+                <span className="text-sm text-gray-400 sm:text-lg sm:ml-2">
+                  ({user.email})
+                </span>
+              )}
```

Comment:
We already filter email address from the API responses if the user doesn't have the `MANAGE_USERS` permission I the API.